app.py

```diff
@@ -183,7 +183,7 @@ def make_es_request(**kwargs):
     return r
 
 
-def es_query(query, index, size):
+def es_query(query, index, size, body={}):
```

Comment:
Probably this is an unintentional use of a mutable default argument, right?

Reply:
My thinking was that since I don't actually manipulate the `body` dict (instead, I call `otherdict.update(body)`) the mutable default argument doesn't matter, but after thinking about it, it would be better to make `body=None` in case of future changes. I'll update this, thanks!